pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-finance-storage</artifactId>
-  <version>7.2.0-SNAPSHOT</version>
+  <version>8.0.0-SNAPSHOT</version>
```

Comment:
Please, do not edit project version mannualy. This will be don in scope of release PR

Reply:
OK, but then how will we know we have to increase the major version when releasing ?

Reply:
Person who will do the release should quickly review the stories and make a decision about release version.